src/SLCore.IntegrationTests/SLCoreTestRunner.cs

```diff
@@ -148,12 +157,17 @@ private void SetUpLocalFolders()
 
         if (Directory.Exists(privateFolder))
         {
-            Directory.Delete(privateFolder, true);
+            DeleteDirectoryForLongPaths(privateFolder);
         }
 
         Directory.CreateDirectory(privateFolder);
         Directory.CreateDirectory(storageRoot);
         Directory.CreateDirectory(workDir);
         Directory.CreateDirectory(userHome);
     }
+
+    private static void DeleteDirectoryForLongPaths(string dirToDelete)
+    {
+        Directory.Delete(@"\\?\" + dirToDelete, true);
```

Comment:
Ideally should be a separate PR as it is an unrelated change. And how does it work?

Reply:
It is related to the integration test, that was failing for me locally. 

The @"\\?\" prefix is a special syntax used to handle long paths in Windows. Normally, Windows paths are limited to 260 characters, but using the @"\\?\" prefix allows you to work with paths up to 32,767 characters.

https://learn.microsoft.com/en-gb/windows/win32/fileio/naming-a-file?redirectedfrom=MSDN

Reply:
Have you tried this? https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=powershell#registry-setting-to-enable-long-paths

Reply:
Yes. Locally I could delete the folders, but the unit tests were failing